client/src/Components/Layouts/SingleCaptains/Popups/viewPopUp.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from "react";
+import { Modal, Table} from "antd";
+
+class View extends Component {
+  state = {
+    visible: this.props.visible,
+    id: this.props.id,
+    information: null
+  };
+
+  handleCancel = e => {
+    this.props.visibleFun("singleCaptain", "viewVisibility")(e);
+  };
+  componentWillReceiveProps(props) {
+    const { visible, id, information } = props;
+    this.setState({ visible, id, information });
+  }
+
+  render() {
+      const columns = [{title: 'اسم الطلبية', dataIndex: 'f1'},
+      {title: 'السعر', dataIndex: 'f2'}]
+    if (this.state.information) {
+      const { information } = this.state;
```

Comment:
@engshorouq If you want, you could swap the order of these lines:
```
const { information } = this.state
if ( information ) {
```
The destructure will work even if information is falsy.